pkg/processor/runtime/python/py/requirements/common.txt

```diff
@@ -1,3 +1,7 @@
 mock==2.0.0
-nuclio-sdk==0.2.0
-msgpack==0.6.1
+
+# TODO: uncomment
+#nuclio-sdk==0.3.0
+
+# TODO: remove
+git+https://github.com/liranbg/nuclio-sdk-py.git@msgpack102
```

Comment:
TODO ?

Reply:
part of REVERTME commit
would be removed once `sdk` is released, uncomment the line above and _then_ merge